contextual_services/contextual_services.model.lkml

```diff
@@ -2,3 +2,33 @@ connection: "bigquery-oauth"
 label: "Contextual Services"
 include: "//looker-hub/contextual_services/explores/*"
 include: "views/*"
+
+
+explore: +event_aggregates {
+  description: "Daily event rollups. This aggregated dataset is faster to query and contains a longer history."
+}
+
+explore: +quicksuggest_clicks {
+  description: "
+    The QuickSuggest click ping is sent when the user clicks on the QuickSuggest link.
+    It shares most of the fields as the QuickSuggest impression ping except that it doesn’t include the user search query, nor the matched keywords.
+    This data is kept for 30 days."
```

Comment:
Can we add this sentence: Click pings may be sent more frequently than impression pings.

Reply:
Can you describe how? Is this because of multiple clicks for the same impression, or because we're somehow not sending an impression ping?

Reply:
We send separate pings for clicks and impressions to facilitate data processing and data sharing with our partners. More details here: https://docs.google.com/document/d/1crGV6l9wXh2ITmvj6tC5taBGM23XcUaf4B89w_cPdwE/edit#bookmark=id.g5v9vhy7sfry

Reply:
Okay, I was misunderstanding. You mean they will be sent with lower latency, is that right?

Reply:
yes, sorry, latency not frequency.